MediaBrowser.Model/Branding/BrandingOptions.cs

```diff
@@ -15,5 +18,21 @@ public class BrandingOptions
         /// </summary>
         /// <value>The custom CSS.</value>
         public string? CustomCss { get; set; }
+
+        /// <summary>
+        /// Gets or sets the splashscreen location on disk.
+        /// </summary>
+        /// <remarks>
+        /// Not served via the API.
+        /// Only used to save the custom uploaded user splashscreen in the configuration file.
+        /// </remarks>
+        [JsonIgnore]
+        public string? SplashscreenLocation { get; set; }
+
+        /// <summary>
+        /// Gets the splashscreen url.
+        /// </summary>
+        [XmlIgnore]
+        public string SplashscreenUrl => "/Branding/Splashscreen";
```

Comment:
New API properties should be marked nullable to maintain backwards compatibility in the SDK's/apiclients

Reply:
This property has no setter. So it doesn't matter if the clients send data in this property or not. It will always have the value `/Branding/Splashscreen`. The serializer doesn't throw an error if you don't send data for this property. So not sending it at all isn't a problem.
For deserializing the json by the clients shouldn't be a problem either since most serializers just ignore json properties they don't know.
So I don't think backwards compatibility is a problem here.

Reply:
I believe this property does need to be nullable as the Kotlin sdk doesn't like properties missing that are marked as non-null

Reply:
Then I'm sorry I misunderstood you @nielsvanvelzen. I thought you were talking about an issue validating the object server-side.